source/css/partners/components/_header.scss

```diff
@@ -29,18 +35,15 @@ $__categories_list_bottom_margin: $gutter/2;
     }
 
     &-list {
-      margin-left: $gutter / 3;
+      // margin-left: $gutter / 3;
```

Comment:
how come this is left in but commented out?

Reply:
Ah, I will just remove actually.